### PR TITLE
improvement(config-graph): add more data to rendered graph nodes

### PR DESCRIPTION
**Pull request by eysi09**

### CODE REVIEW

File: garden-service/src/config-graph.ts [hunk 1]
```diff
@@ -524,9 +552,47 @@ export class DependencyGraphNode {
...
+          ...renderedNode,
+        }
+      case "run":
+        return {
+          type,
+          taskName: this.name,
+          ...renderedNode,
+        }
+      case "test":
+        const { testName } = parseTestKey(this.name)
```
Comment (thsig) [inline garden-service/src/config-graph.ts]: Maybe I'm misunderstanding something here, but couldn't this be read directly from `this.testName`?

File: garden-service/src/config-graph.ts [hunk 2]
```diff
@@ -576,6 +642,15 @@ function nodeKey(type: DependencyGraphNodeType, name: string) {
   return `${type}.${name}`
 }
 
+function testKey(moduleName: string, testName: string) {
+  return `${moduleName}.${testName}`
+}
+
+function parseTestKey(key: string) {
```
Comment (thsig) [inline garden-service/src/config-graph.ts]: If the above comment is valid, then this function can be removed.

File: garden-service/src/config-graph.ts (hunk 1 above)
Comment (eysi09) [reply garden-service/src/config-graph.ts]: There is no `this.testName`.

Comment (eysi09) [reply garden-service/src/config-graph.ts]: ...but perhaps the function is a little superfluous.

**Pull request merged**